core/lib/resources/presence/presence_manager.js

```diff
@@ -243,16 +234,16 @@ PresenceManager.prototype.processRedisEntry = function(message, callback) {
       message.sentry = userId + '.' + socketId;
 
       // Publish fake entry, autopub will renew it
-      sentry.publishKeepAlive({ name: message.sentry, save: false });
+      sentry._keepAlive({ name: message.sentry, save: false });
     }
```

Comment:
remove this entire if !message.sentry section. This is for supporting the old style servers but we no longer have them.